doc/classes/AudioEffectEQ10.xml

```diff
@@ -1,8 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectEQ10" inherits="AudioEffectEQ" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a 10 bands equalizer audio effect to an Audio bus. Gives you control over frequencies from 31 Hz to 16000 Hz.
```

Comment:
"10 band"

Reply:
"10-band" even, if I remember my English lessons correctly :)

Reply:
I think it's interchangeable, but yeah "10-band" looks better.